django/applications/catmaid/management/commands/catmaid_cache_update_worker.py

```diff
@@ -140,10 +140,10 @@ def run_available_tasks(self):
             except Exception as e:
                 logger.exception('Error in %r: %r.', e.job, e, extra={
                     'data': {
-                        'job': e.job.to_json(),
+                        'job': e.job.to_json(), # FIXME: In this case, job will be None
```

Comment:
logic here is not quite right, IIRC

Reply:
That's true and it also refers to another `queue` implementation than the list I use. This function (just like the same in `catmaid_spatial_update_worker.py`) isn't used at the moment. Therefore, I would be in favor removing these functions altogether along with the commented out call sites. I copy the current version to a branch to investigate later what the intention was originally.